src/main/webapp/vendor/startbootstrap/index.html

```diff
@@ -18,9 +18,11 @@
   <!-- Custom styles for this template-->
   <link href="https://cdn.jsdelivr.net/npm/startbootstrap-sb-admin-2@4.1.1/css/sb-admin-2.min.css" rel="stylesheet">
 
+  <script src="script.js"></script>
+  
 </head>
 
-<body id="page-top">
+<body id="page-top" onload="getOptions()">
```

Comment:
`onload` is apparently not very secure. Is it possible here to call `getOptions()` in js instead?